app/components/post_list/post/body/index.tsx

```diff
@@ -142,7 +142,7 @@ const Body = ({
             <Message
                 highlight={highlight}
                 isEdited={isEdited}
-                isPendingOrFailed={isPendingOrFailed}
+                isPendingOrFailed={isFailed}
```

Comment:
0/5 if we want to change the prop name

Reply:
Before addressing these changes, I will wait for the UX input on whether we should continue this path or not.